### ISSUE TITLE
Inexplicit likes ...

### ISSUE THREAD

**Issue by digiloo** [pH7Software/pH7-Social-Dating-CMS#736]
I suggest that it should be clearer what exactly is liked (album, picture, member) then you
click on **Like.**

E.g see the below screenshot:
https://user-images.githubusercontent.com/1325411/35779585-68f0d5fc-09c7-11e8-91eb-bf793fcfab6e.png. I  guess that _Like deniz_ should be e.g _Like Maldives! album_ Or?

/ Digiloo

**Comment by pH-7** [comment]
You're right. Makes sense by now, when looking to your screenshot. Something definetly has to be updated in the `Design::like()` to take this into account.

Further information, where this happends at the moment: https://github.com/pH7Software/pH7-Social-Dating-CMS/blob/dc9b5ffffc560d89d0631de61eab4c345e9dbe5d/_protected/framework/Layout/Html/Design.class.php#L785

**Comment by pH-7** [comment]
Thanks for the suggestion.

I reviewed this during backlog cleanup. This request is valid but outside the current 18.x delivery scope, so I’m closing it as not planned for now to keep the tracker focused on active work.

If you want to push this forward, please open a new proposal with a minimal implementation plan (scope, DB/API impact, migration notes, and backward-compatibility expectations).

Take care and thank you for using pH7 Social Dating Builder,

[Pierre-Henry](https://github.com/pH-7)
